src/TuDa.CIMS.Api/Controllers/AssetItemController.cs

```diff
@@ -76,4 +74,20 @@ public async Task<IActionResult> RemoveAsync(Guid id)
             err => BadRequest(err)
         );
     }
+
+    /// <summary>
+    /// Returns a paginated response of AssetItems based on the provided user parameters.
+    /// If the operation is successful, returns a 200 OK response.
+    /// If an error occurs during the operation, an appropriate error response is returned.
+    /// </summary>
+    /// <param name="userParams"></param>
+    /// <returns></returns>
+    [HttpGet]
```

Comment:
🔧 This needs to be unique so `GetAll` and this does not clash
```suggestion
    [HttpGet("paginated")]
```